### PR TITLE
Website links for Zope->Jahia transition; \textnormal rendering fix

### PR DESCRIPTION
**Pull request by sean-lockwood**
**DO NOT MAKE THESE CHANGES LIVE UNTIL THE NEW WEBSITE IS UP AND RUNNING!**
The Zope --> Jahia transition is currently scheduled for Monday, June 17, 2019.  I updated links, but most cannot be tested until after the new site goes live.

Here are a few links I haven't updated but probably need to be modified:
 - http://www.stsci.edu/hst/observatory/crds/SIfileInfo/pysynphottables/index_phoenix_models_html
 - https://aeon.stsci.edu/ssb/trac/astrolib/ticket/218
 - http://ssb.stsci.edu/cdbs/tarfiles/synphot1.tar.gz
 - http://ssb.stsci.edu/cdbs/tarfiles/synphot2.tar.gz
 - http://ssb.stsci.edu/cdbs/tarfiles/synphot3.tar.gz
 - http://ssb.stsci.edu/cdbs/tarfiles/synphot4.tar.gz
 - http://ssb.stsci.edu/cdbs/tarfiles/synphot5.tar.gz
 - http://ssb.stsci.edu/cdbs/tarfiles/synphot6.tar.gz
 - http://www.stsci.edu/hst/observatory/crds/documents/TIR-CDBS-2012-01.pdf
 - http://ssb.stsci.edu/perry/pydatatut.pdf
 - http://ssb.stsci.edu/tsr/2009_01/
 - http://www.stsci.edu/hst/HST_overview/documents/synphot/hst_synphotTOC.html

Things I updated but are worthy of further consideration:
 - ssb.stsci.edu/cdbs --> ftp site
 - http://www.sdss.org/dr3/instruments/imager/#filters --> dr7
 - http://www.sdss.org/dr3/algorithms/fluxcal.html     --> dr7

Aside from links, I updated these things:
 - A few pages were put into Unix LF format for consistency.
 - `\textnormal{}` --> `\mathrm{}` throughout to fix Sphinx rendering of math.

### CODE REVIEW

**Comment by sean-lockwood** [comment]
There shouldn't have been any code changes, so I'm not sure why Travis is failing.

File: doc/source/appendixa.rst [hunk 1]
```diff
@@ -103,7 +103,7 @@ Considering that the entire atlas occupies close to 70 MB of disk space,
 many applications could be satisfied by a copy of the solar metallicity
 spectra only (Table 2 of the README file).
 See
-`Kurucz 1993 atlas README file <http://ssb.stsci.edu/cdbs/grid/k93models/AA_README>`_
+`Kurucz 1993 atlas README file <ftp://ftp.stsci.edu/cdbs/grid/k93models/AA_README>`_
```
Comment (pllim) [inline doc/source/appendixa.rst]: Please revert all `ssb.stsci.edu -> ftp`. Some Macs don't support FTP anymore. The HTTP is a workaround.

**Comment by pllim** [comment]
@tddesjardins , were throughput files for obsmode `acs,sbc,f122m` updated recently? The test failure is because this obsmode produce new result now. Is that expected?

**Comment by pllim** [comment]
@sean-lockwood , thanks, but is it possible to break this PR down into several smaller PRs? For example, the `textnormal` stuff is uncontroversial enough and does not need to wait for Zope to die to be merged.

I would like the Zope one to be isolated by itself, if possible. Thanks!

**Comment by tddesjardins** [comment]
@pllim yes, new throughput tables were delivered for SBC. Check out issue ACS-14 on JIRA for more information.

File: doc/source/appendixa.rst (hunk 1 above)
Comment (sean-lockwood) [reply doc/source/appendixa.rst]: @pllim Will that http link exist after Monday?

**Comment by pllim** [comment]
Re: Failing test -- It is fixed in #114 . A rebase would make it go away for your PR(s).

**Comment by sean-lockwood** [comment]
The `\textnormal` stuff is now in PR #115.

File: doc/source/appendixa.rst (hunk 1 above)
Comment (sean-lockwood) [reply doc/source/appendixa.rst]: The link text [here](https://pysynphot.readthedocs.io/en/latest/how_works.html#the-pysynphot-database) to `ssb.stsci.edu/cdbs` does not return a valid site.  Did you have an alternative?

Comment (pllim) [reply doc/source/appendixa.rst]: Just `ssb.stsci.edu/cdbs` is not supposed to work. You need the URL to the exact file, like http://ssb.stsci.edu/cdbs/grid/k93models/AA_README .

**Pull request closed without merging**